plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java

```diff
@@ -413,4 +432,8 @@ protected String getColumnComment(String tableName, String columnName)
     protected abstract void dropTableFromMetastore(String tableName);
 
     protected abstract String getMetadataLocation(String tableName);
+
+    protected abstract String generateTableLocation(String tableName);
```

Comment:
I expect that "generate table" generates a new table name with random suffix or something, but it's just formatting a table name. 

GCS and Glue tests already have `schemaUrl` and `schemaPath` methods. Can we rename them to the same method name and add a method for schema location to here? 

By the way, can we get `local` schema properties from `SHOW CREATE SCHEMA` and avoid adding this method?

Reply:
we can not execute `SHOW CREATE SCHEMA` because the schema is not created and not existed. I will rename the method to match the existing method.